crates/issue/src/lib.rs

```diff
@@ -471,17 +479,13 @@ impl<T: Config> Pallet<T> {
                 return Err(Error::<T>::TimeNotExpired.into());
             };
 
-        if ext::vault_registry::is_vault_liquidated::<T>(&issue.vault)? {
```

Comment:
Is this change necessary?

Reply:
This is the primary change to slash the griefing collateral to the treasury, or do you mean specifically when we return slashed griefing collateral if the vault is liquidated?

Reply:
> do you mean specifically when we return slashed griefing collateral if the vault is liquidated?

Yes that's what I mean

Reply:
I would prefer for simplicity to always transfer to the treasury, also even if the Vault is liquidated it still blocked minting this way

Reply:
For self-issues yes I agree it makes sense. When the requester is not the vault, though, it's not so clear to me. If a user sees that a vault is liquidated before it gets the chance to send the btc, isn't it reasonable to allow the user to cancel without confiscating the griefing collateral?